src/burnchains/burnchain.rs

```diff
@@ -2348,156 +2326,6 @@ pub mod tests {
         }
     }
 
-    #[test]
-    fn test_filter_block_commits_with_same_VRF_key() {
```

Comment:
Rather than deleting this test, should we update the assertions to accept multiple block commits?

Reply:
The only thing this test did was check to see that the now-removed `test_filter_block_commits_with_same_VRF_key()` function actually did the requisite filtering.  It didn't process or insert the block-commits into a database or anything that would make the kinds of checks you're asking for possible.

Instead of restoring this check, would it make more sense to expand the `test_check()` test in `leader_block_commit.rs` to verify that two block-commits can reference the same VRF key and be accepted?

Reply:
> Instead of restoring this check, would it make more sense to expand the test_check() test in leader_block_commit.rs to verify that two block-commits can reference the same VRF key and be accepted?

Yep, that makes sense.

Reply:
Addressed in c9285de